tests/Functional/ContentProjectRenderTest.php

```diff
@@ -107,6 +107,18 @@ public function testProjectRendering(): void {
         ],
       ],
       'oe_reference' => 'Project reference',
+      'oe_project_locations' => [
```

Comment:
It looks a little bit different than on the mockup. Address field doesn't provide regions "Bavaria", or "Brussels".

<img width="761" alt="Screenshot 2020-08-27 at 12 51 46" src="https://user-images.githubusercontent.com/1183951/91435577-31a2fb00-e86f-11ea-82f4-bf7b54a6344f.png">